src/main/webapp/js/user-page-loader.js

```diff
@@ -48,18 +48,18 @@ function fetchAboutMe(){
 }
 
 /**
- * Shows the message form if the user is logged in and viewing their own page.
+ * Shows the message form if the user is logged in.
  */
-function showMessageFormIfViewingSelf() {
+function showMessageFormIfLoggedIn() {
   fetch('/login-status')
       .then((response) => {
         return response.json();
       })
       .then((loginStatus) => {
-        if (loginStatus.isLoggedIn &&
-            loginStatus.username == parameterUsername) {
+        if (loginStatus.isLoggedIn) {
           const messageForm = document.getElementById('message-form');
-          messageForm.classList.remove('hidden');
+        messageForm.action = '/messages?recipient=' + parameterUsername;
```

Comment:
Hmmm. Is this formatted correctly? If it's not, I have no clue why the linter did not catch this 😆